### ISSUE TITLE
Formatting license

### ISSUE THREAD

**Issue by matbesancon** [goualard-f/MicroFloatingPoints.jl#7]
The license header has some alignment issues:

```
# MicroFloatingPoints --
#
#	Copyright 2019--2023 University of Nantes, France.
#
#	This file is part of the MicroFloatingPoints library.
#
#	The MicroFloatingPoints library is free software; you can redistribute it and/or
# modify it under the terms of the GNU Lesser General Public License as published
# by the Free Software Foundation; either version 3 of the License, or (at your
#	option) any later version.
#	
#	The MicroFloatingPoints library is distributed in the hope that it will be useful,
# but	WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
#	or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
#	for more details.
#	
#	You should have received copies of the GNU General Public License and the
#	GNU Lesser General Public License along with the MicroFloatingPoints Library.
# If not,	see https://www.gnu.org/licenses/.
```

**Comment by matbesancon** [comment]
In the context of:
https://github.com/openjournals/joss-reviews/issues/7050

**Comment by goualard-f** [comment]
Corrected by commit [a8b0dba](https://github.com/goualard-f/MicroFloatingPoints.jl/commit/a8b0dba053506ff2bdc46249bfb3ae6e6d96980b)
